src/engine/Engine.cpp

```diff
@@ -210,7 +210,10 @@ bool Engine::solve( unsigned timeoutInSeconds )
                     if ( _tableau->getBasicAssignmentStatus() !=
                          ITableau::BASIC_ASSIGNMENT_JUST_COMPUTED )
                     {
-                        printf( "Before declaring SAT, recomputing...\n" );
+                        if ( _verbosity > 0)
```

Comment:
Please add a space: "> 0)" --> "> 0 )"

Apart from that, LGTM - you can go ahead and merge.

Reply:
Added the space, I'll merge this in.